src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/HomePage.java

```diff
@@ -0,0 +1,171 @@
+package com.marizueva.laboratory.hw3.fluentpageobject.pages;
+
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends CommonPageElements {
+
+
+    @FindBy(xpath = "//*[@class='benefit-icon'][1]")
+    WebElement displayedBenefitImage1;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-custom'][1]")
+    WebElement displayedBenefitImage2;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-multi'][1]")
+    WebElement displayedBenefitImage3;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-base'][1]")
+    WebElement displayedBenefitImage4;
```

Comment:
Why do you use indexes in the xpath? This is not good practice because elements could be added before with the same class and your locator will be incorrect

Reply:
all xpath with indexes are elominated